CHANGELOG.md

```diff
@@ -1,17 +1,20 @@
 # Developing
+
 1.  Add FileEmitter and RotatingFileEmitter to xylog.
 2.  Xylog now can be easy to use key-value fields and extra values.
 
 # V1.0.1
-1. Fix bugs.
-2. Refactor the design of xylog Handler.
-3. Add github workflow to test.
-4. Write unittest for all modules.
-5. Xycond asserts a Error instead of string.
 
+1.  Fix bugs.
+2.  Refactor the design of xylog Handler.
+3.  Add github workflow to test.
+4.  Write unittest for all modules.
+5.  Xycond asserts a Error instead of string.
```

Comment:
Codacy has a fix for the issue: [Expected: 1; Actual: 2](https://app.codacy.com/gh/xybor/xyplatform/pullRequest?prid=10050498)
```suggestion
5. Xycond asserts a Error instead of string.
```